app.rb

```diff
@@ -63,10 +63,6 @@
           pull_request.add_comment(message)
         end
       elsif !issue.rejected?
-        if project.name == 'Katello' && issue.release == 'Katello Backlog'
-          issue.set_release(nil)
-        end
```

Comment:
@ekohl okay, we've moved our process to use the triaged flag, can you update this to set triaged to 'false' instead of this?

Reply:
Should this still be limited to Katello or does Foreman follow a similar pattern?

Reply:
I'm not sure, @tbrisker @GregSutcliffe have you guys moved to use the triaged flag yet?  and if so would you like for issues to 'untriage' when a pr gets opened?

Reply:
Having thought about it, we can just make a boolean in the config if this behavior is desired. That makes it trivial to enable it for other repos too.